operations/mimir/ruler.libsonnet

```diff
@@ -44,12 +44,14 @@
 
   ruler_deployment:
     if $._config.ruler_enabled then
-      deployment.new('ruler', 2, [$.ruler_container]) +
+      local name = 'ruler';
```

Comment:
Shall we follow querier vs newQuerierDeployment pattern here (and for distributor too)?

Reply:
Maybe. So far we did on-demand (when needed).